include/sys/fs/zfs.h

```diff
@@ -184,6 +184,8 @@ typedef enum {
 	ZFS_PROP_REMAPTXG,		/* not exposed to the user */
 	ZFS_PROP_SPECIAL_SMALL_BLOCKS,
 	ZFS_PROP_IVSET_GUID,		/* not exposed to the user */
+	ZFS_PROP_REDACTED,
+	ZFS_PROP_REDACT_SNAPS,
```

Comment:
What is the impact of changing the zfs_prop_t values?  Is it just "external consumers" (per the comment above), i.e. libzfs consumers?  AFAIK, we don't make any guarantees about the libzfs interface changing.  If we are going to make a "best effort" attempt to not change libzfs interfaces, that should apply to function signatures as well.

These values are not stored on disk, and the user/kernel interface uses the prop names, not zfs_prop_t (I think).

Seems like this is something we could improve in the future, by having the libzfs interface use the prop names.